#ComparisonScores.msg

std_msgs/Header header
int32 id
float32 hoc_distance_score
float32 pose_distance_score
